lib/registrant.rb

```diff
@@ -0,0 +1,35 @@
+class Registrant
+  attr_reader :name, :age, :permit, :license_data
+
+  def initialize(name, age, permit = false)
+    @name = name
+    @age = age
+    @permit = permit
+    @license_data = {
+      written: false,
+      license: false,
+      renewed: false
+    }
+  end
+  
+  def permit?
+    @permit
+  end
+
+  def earn_permit
+    @permit = true
+  end
+
+  def written?
```

Comment:
Interesting choice to break these apart into their own methods. I like it! I will peek in the tests to make sure all of these helper methods have their own tests. :)